server/viame_server/viame_detection.py

```diff
@@ -128,7 +127,9 @@ def export_detections(self, folder):
 
         filename = ".".join([file["name"].split(".")[:-1][0], "csv"])
 
-        csv_string = viame.export_tracks_as_csv(file)
+        imageFiles = [f['name'] for f in Folder().childItems(folder, filters={"lowerName": {"$regex": r'^\.jpg|\.jpeg|\.png' }}).sort("lowerName")]
```

Comment:
Import `webValidImageFormats` then build the search from that set:

```suggestion
        imageRegex = re.compile("^\." + "|\.".join(webValidImageFormats))
        imageFiles = [f['name'] for f in Folder().childItems(folder, filters={"lowerName": {"$regex": imageRegex}}).sort("lowerName")]
```